lametro/events.py

```diff
@@ -1,10 +1,31 @@
 import datetime
+import logging
 
 from legistar.events import LegistarAPIEventScraper
 from pupa.scrape import Event, Scraper
 from legistar.base import LegistarScraper
 
 
+LOGGER = logging.getLogger(__name__)
+
+class UnmatchedEventError(Exception):
+    def __init__(self, events):
+        if type(events) is dictionary:
+            message = "Can't find companion for Event \
+                                    {0} at {1} on {2} - {3}" \
+                                    .format(events['EventId'], events['EventTime'], \
+                                    events['EventDate'], EventInSiteURL['EventInSiteURL'])
+        elif type(events) is list:
+            message = ''
+            for event in events:
+                temp = "Can't find companion for Event \
+                                        {0} at {1} on {2} - {3} \n" \
+                                        .format(event['EventId'], event['EventTime'], \
+                                        event['EventDate'], event['EventInSiteURL'])
+                message += temp
```

Comment:
you might consider adding newlines (`\n`) between messages as you build this up!

Reply:
i did! https://github.com/opencivicdata/scrapers-us-municipal/pull/245/files/859bb74f23745edb2c3a050b2451b717c6eeaf10#diff-bc7568ae7d929ea23440d86969ff9fceR22

Reply:
o word!